packages/api-reference/src/features/Search/SearchButton.vue

```diff
@@ -63,11 +64,9 @@ function handleClick() {
     role="search"
     type="button"
     @click="handleClick">
-    <ScalarIcon
+    <ScalarIconMagnifyingGlass
```

Comment:
Added this here to test tree shaking and see if the bundle explodes

Reply:
![image](https://github.com/user-attachments/assets/12cdbf32-69f8-4291-9b54-96bde5dd8a2f)


![Google Chrome-2025-04-16-22-07-19@2x](https://github.com/user-attachments/assets/4803137e-b678-4a88-aefa-d1629c500ad9)